gensrc/thrift/PlanNodes.thrift

```diff
@@ -365,14 +365,19 @@ struct THdfsScanRange {
     // delete columns slots like iceberg equality delete column slots
     21: optional list<Types.TSlotId> delete_column_slot_ids;
 
+    22: optional bool use_iceberg_jni_metadata_reader
+
+    // for metadata table split (eg: iceberg manifest file bean)
+    23: optional string serialized_split
+
     // whether to use JNI scanner to read data of kudu table
-    22: optional bool use_kudu_jni_reader
```

Comment:
https://github.com/StarRocks/starrocks/blob/main/gensrc/thrift/PlanNodes.thrift#L375  seq need to be consistent with main